php/libraries/Utility.class.inc

```diff
@@ -150,6 +151,39 @@ class Utility
         return $list;
     }
 
+    /**
+     * Returns a list of sites by site IDs from the database
+     *
+     * @param array $siteIDs a list of the study center IDs
+     *
+     * @return array an associative array("center ID" => "site name")
+     */
+    static function getSiteListByIDs(array $siteIDs): array
+    {
+        $factory = NDB_Factory::singleton();
+        $DB      = $factory->database();
+
+        $sites  = array();
+        $params = array();
+        $values = array();
+        foreach ($siteIDs as $key => $siteID) {
+            $para          = "cid$key";
+            $params[]      = ":$para";
+            $values[$para] = $siteID;
+        }
+
+        $centerLists = $DB->pselect(
```

Comment:
I'm not a big fan of switching between `site` and `center` just choose one and use it for variable names

Reply:
I ignore why initially that center vs site was named in that way, but I changed center to site.